components/hybrid_engine/src/observer/load_eviction.rs

```diff
@@ -133,6 +133,31 @@ impl LoadEvictionObserver {
 impl Coprocessor for LoadEvictionObserver {}
 
 impl QueryObserver for LoadEvictionObserver {
+    fn pre_exec_query(
+        &self,
+        ctx: &mut ObserverContext<'_>,
+        reqs: &[kvproto::raft_cmdpb::Request],
+        _: u64,
+        _: u64,
+    ) -> bool {
+        reqs.iter().for_each(|r| {
```

Comment:
```suggestion
        for r in reqs {
```

Prefer for loop for better readability.

Reply:
I suppose rust should prefer functional than procedural.

Reply:
+1